### ISSUE TITLE
*Datetable* - Error for Guest User - Unauthenticated

### ISSUE THREAD

**Issue by bryan3200** [alexed1/LightningFlowComponents#1331]
### Steps to reproduce

1. Load flow that has the datatable on it that works in Community and internal to a page that guest user can access.
2. Open link to the page and you get an error when it tries to render the datatable.
See error:  Cannot read properties of undefined (reading 'split')

### Screenshots
![image](https://user-images.githubusercontent.com/106887146/213897054-67ac93e3-ee16-462e-a467-eccd6b2d2e49.png)

**Comment by ericrsmith35** [comment]
What version of Datatable are you running?

**Comment by BDOGG32** [comment]
Version 4.0.12

**Comment by ericrsmith35** [comment]
Does the flow work with authenticated users?  Is it set to run in System Mode?

**Comment by bryan3200** [comment]
Yes it does work with Authenticated Users.  It is running in system mode.

**Comment by ericrsmith35** [comment]
I think this may be a security issue.  Do other flows with standard components work?

**Comment by bryan3200** [comment]
Yes they work.  I can get it to show everything on the screen but the Datatable component.

**Comment by ericrsmith35** [comment]
It may have to do with how I'm trying to generate the site url.  Can you run the flow and show me what appears in the browser console view? (CTRL-SHIFT-I), select the Console tab.

**Comment by bryan3200** [comment]
Here is what I get:

![image](https://user-images.githubusercontent.com/106887146/215286120-469f8d20-0ae9-498e-9b4f-0d7cf33b8088.png)

**Comment by bryan3200** [comment]
I am getting that behavior with a LWR site.  If I use aura site it gives me this:

![image](https://user-images.githubusercontent.com/106887146/215286236-fefaf7f9-bd63-4d61-b81d-7319d92fefee.png)

**Comment by ericrsmith35** [comment]
It definately looks like LWR sites may have an issue here.  What object are you using for the Datatable?

**Comment by bryan3200** [comment]
ContentDistrubution - We have public links to documents that we want guest users to have access to in a table.

**Comment by ericrsmith35** [comment]
Does this work with the new standard Data Table component?

**Comment by bryan3200** [comment]
I have not tried that.  But I will give it a shot.

**Comment by bryan3200** [comment]
I have tried and it errors out on the LWR site.  I get this for the standard Data Table:  Something went wrong with the "Voting_Files_table" screen component on the "Cast Ballot Flow" flow. Contact your Salesforce admin about this error. LWR3008: Error loading https://vforce--vmxdev2.sandbox.my.site.com/virtualforms/webruntime/component/latest/prod/en-US/dxp_flowruntime%2Fdatatable

![image](https://user-images.githubusercontent.com/106887146/215343230-18140206-f99a-42fa-88af-22b71b6ee3d6.png)


With that said I tried it on the aura framework site it does work and shows the data table with Content Distribution.
